options/posix/generic/posix_stdlib.cpp

```diff
@@ -217,6 +217,13 @@ char *realpath(const char *path, char *out) {
 
 	// If the path is relative, we have to preprend the working directory.
 	if(path[0] == '/') {
+		if(strlen(path) == 1) {
+			if(!out)
+				out = reinterpret_cast<char *>(getAllocator().allocate(2));
+			strcpy(out, path);
+			return out;
+		}
```

Comment:
I don't think that this is correct. This is supposed to handle `/` inputs, right? However, `//` is equivalent to `/` and that is not handled.